### ISSUE TITLE
Conditional formatting

### ISSUE THREAD

**Issue by pecrazouteisau** [trey-wallis/obsidian-dashboards#586]
### Problem

I can't visually see differences between values (numbers) in cells.

### Solution

Conditional formatting setting per-column\per-row\per-array-of-cells (which values are used as a context for min\max\X%). Set cell background colors for min, max, X% and X, place each cell number on this dynamic gradient and resolve what background color should this cell have.

### Additional details

Basically: https://blog.coupler.io/wp-content/uploads/2021/01/0-conditional-formatting-google-sheets.png
